viofs/svc/virtiofs.cpp

```diff
@@ -143,10 +144,23 @@ struct VIRTFS
 };
 
 VIRTFS::VIRTFS(ULONG DebugFlags, const std::wstring& MountPoint)
-    : DebugFlags{ DebugFlags }, MountPoint{ MountPoint }
+    : DebugFlags{ DebugFlags }, MountPoint{ MountPoint }, Tag{ Tag }
 {
 }
 
+template <typename EF>
+class scope_exit
```

Comment:
Why you manually add this class? C++ has it https://en.cppreference.com/w/cpp/experimental/scope_exit, any reason to don't use library?

Reply:
Because it is an experimental feature, and it seems to be unavailable in MSVC. When it become available custom variant can be easily replaced to variant from library.